index.css

```diff
@@ -0,0 +1,64 @@
+
+
+*{
+    margin: 0%;
+    padding: 0%;
+    box-sizing: border-box;
+}
+h1{
```

Comment:
The typical formatting for CSS is to use a paragraph between different css rules, and a space between the selector and opening brace, for example:

```
...
}

h1 {
...
```

-1